src/RepoAuditor/Plugins/GitHub/ClassicBranchProtectionRequirements/Impl/ClassicEnableRequirementImpl.py

```diff
@@ -19,20 +19,19 @@ class ClassicEnableRequirementImpl(EnableRequirementImpl):
     def __init__(
         self,
         name: str,
-        default_value: bool,
         dynamic_arg_name: str,
         github_settings_section: str,
         github_settings_value: str,
         get_configuration_value_func: Callable[[dict[str, Any]], Optional[bool]],
         rationale: str,
         subject: Optional[str] = None,
         *,
+        default_value: bool,

```

Comment:
In this case, I'd like to suppress the warning rather than changing the order of the arguments. The \`ClassicEnableRequirementImpl\` and \`ClassicValueRequirementImpl\` classes are (potentially over-) complicated. When writing them, I took care to ensure that the parameters appeared in the same order for both of them to make it a bit easier for derived classes to use. With this change, the parameter ordering has changed and now the person using the base classes needs to remember to use a specific calling style in each case.